spacy_llm/tasks/lemma/registry.py

```diff
@@ -0,0 +1,47 @@
+from typing import Optional, Type
+
+from ...registry import registry
+from ...ty import ExamplesConfigType, FewshotExample, TaskResponseParser
+from .task import LemmaTask
+from .util import DEFAULT_LEMMA_TEMPLATE_V1, LemmaExample, parse_responses_v1
+
+
+@registry.llm_misc("spacy.LemmaParser.v1")
+def make_lemma_parser() -> TaskResponseParser:
+    return parse_responses_v1
+
+
+@registry.llm_misc("spacy.LemmaFewshotExample.v1")
+def make_lemma_fewshot_example() -> TaskResponseParser:
+    return parse_responses_v1
```

Comment:
Are these supposed to be the same?

Reply:
The latter is redundant, forgot to remove it. Fixed in https://github.com/explosion/spacy-llm/pull/242/commits/047a496ca863c972e2a35600b6492960cac81706.